spec/fixtures/hiera/default.yaml

```diff
@@ -168,4 +168,4 @@ nebula::jdk_version: '11'
 
 nebula::profile::kubelet::kubelet_version: default.invalid
 
-nebula::profile::kubernetes::apt::location: 'https://pkgs.k8s.io/core:/stable:/vX.YZ/deb/'
+nebula::profile::kubernetes::apt::location: ['https://pkgs.k8s.io/core:/stable:/vX.YZ/deb/']
```

Comment:
We should avoid changes that will break existing keys in heiradata unless there is a strong reason to do so.

I'm fairly sure we don't need multiple values here.